bluepyopt/ephys/create_acc.py

```diff
@@ -0,0 +1,348 @@
+        'Nap': {'globals': None, 'ranges': ['gbar', 'g', 'ina']},
+        'SK': {'globals': None, 'ranges': ['gbar', 'ik']}},
+    bbp={
+        'CaDynamics_E2': {'globals': None,
+                        'ranges': ['decay', 'gamma', 'minCai',
+                                   'depth', 'initCai']},
+        'Ca_HVA': {'globals': None, 'ranges': ['gCa_HVAbar']},
+        'Ca_LVAst': {'globals': None, 'ranges': ['gCa_LVAstbar']},
+        'Ih': {'globals': None, 'ranges': ['gIhbar']},
+        'Im': {'globals': None, 'ranges': ['gImbar']},
+        'K_Pst': {'globals': None, 'ranges': ['gK_Pstbar']},
+        'K_Tst': {'globals': None, 'ranges': ['gK_Tstbar']},
+        'NaTa_t': {'globals': None, 'ranges': ['gNaTa_tbar']},
+        'NaTs2_t': {'globals': None, 'ranges': ['gNaTs2_tbar']},
+        'Nap_Et2': {'globals': None, 'ranges': ['gNap_Et2bar']},
+        'SK_E2': {'globals': None, 'ranges': ['gSK_E2bar']},
+        'SKv3_1': {'globals': None, 'ranges': ['gSKv3_1bar']}},
+    default={
+        'exp2syn': {'globals': None, 'ranges': ['tau1', 'tau2', 'e']},
+        'expsyn': {'globals': None, 'ranges': ['tau', 'e']},
+        'expsyn_stdp': {'globals': None,
+                        'ranges': ['tau', 'taupre', 'taupost', 'e',
+                                   'Apost', 'Apre', 'max_weight']},
+        'gj': {'globals': None, 'ranges': ['g']},
+        'hh': {'globals': None,
+               'ranges': ['gnabar', 'gkbar', 'gl', 'el', 'q10']},
+        'kamt': {'globals': ['minf', 'mtau', 'hinf', 'htau'],
+                 'ranges': ['gbar', 'q10']},
+        'kdrmt': {'globals': ['minf', 'mtau'],
+                  'ranges': ['gbar', 'q10', 'vhalfm']},
+        'nax': {'globals': None, 'ranges': ['gbar', 'sh']},
+        'nernst': {'globals': ['R', 'F'], 'ranges': ['coeff']},
+        'pas': {'globals': ['e'], 'ranges': ['g']}}
+)
+
+
+def _find_mech_and_convert_param_name(param, mechs):
+    """Find a parameter's mechanism and convert parameter name to Arbor convention"""
+    mech_suffix_matches = numpy.where([param.name.endswith("_" + mech)
+                                       for mech in mechs])[0]
+    if mech_suffix_matches.size == 0:
+        return None, Location(name=_nrn2arb_var_name(param.name),
+                              value=_nrn2arb_var_value(param)) # TODO: adapt for Range
+    elif mech_suffix_matches.size == 1:
+        mech = mechs[mech_suffix_matches[0]]
+        name = param.name[:-(len(mech)+1)]
+        return mech, Location(name=_nrn2arb_var_name(name),
+                              value=_nrn2arb_var_value(param)) # TODO: adapt for Range
+    else:
+        raise RuntimeError("Parameter name %s matches multiple mechanisms %s " %
+                            (param.name, repr(mechs[mech_suffix_matches])))
+
+
+def _arb_convert_params_and_group_by_mech_global(params, channels):
+    """Group global parameters by mechanism and rename them to Arbor convention"""
```

Comment:
To avoid confusion between global parameters here and GLOBAL nmodl parameters, maybe it is better to use `cell-wide parameters` in the comment?

Reply:
I would try to remain consistent with BluePyOpt's (`create_hoc`) nomenclature. I'm trying to use global/local suffixes to label BluePyOpt locations and when referring to NMODL specifically prepend `nmodl_` to the name. `cell-wide` would be a new name and, thus, also requires refactoring other modules.